api/routes/project.py

```diff
@@ -78,19 +78,23 @@ async def update_project(
     )
 
 
-@router.delete('/{project}', operation_id='deleteProject')
-async def delete_project(
+@router.delete('/{project}', operation_id='deleteProjectData')
```

Comment:
Just noting this is technically a breaking change, but I don't think this method is used so should be fine.

Reply:
Definitely, I didn't find another reference when I looked :)